data-es6.js

```diff
@@ -16047,13 +16047,13 @@ exports.tests = [
         firefox36: {
           val: false,
           note_id: 'array-prototype-iterator',
-          note_html: 'Available as <code>Array.prototype[Symbol.iterator]</code>'
+          note_html: 'Available from Firefox 36 up to 47 as <code>Array.prototype[Symbol.iterator]</code>'
```

Comment:
why only up to v47? `Array.prototype[Symbol.iterator]` is what the spec demands.

Reply:
Because this note will appear only up to 47, for 48 the feature is simply 'true'.  But if this is what the spec demands, can't we remove this note from 36...47?

Reply:
What feature test does this note ID appear on?

Reply:
Only this one.  It has a lot of notes, 4 of them for different Firefox versions.